client/api/python/heketi/heketi.py

```diff
@@ -54,3 +56,535 @@ def hello(self):
         r = requests.get(self.host + uri, headers=headers)
         return r.status_code == requests.codes.ok
 
+
+class Cluster(object):
+    """ Class to run cluster operations """
+
+    def __init__(self, client):
+        self.client = client
+
+    def create(self):
```

Comment:
So when they become part of the same class.. change the names to <endpoint>_<api>, for example `cluster_create()`